ui/apps/platform/src/Containers/NetworkGraph/common/AdvancedFlowsFilter/types.ts

```diff
@@ -8,6 +10,7 @@ export type FilterValue = Directionality | Protocols | Ports;
 
 export type AdvancedFlowsFilterType = {
     directionality: Directionality[];
+    flows: Flow[];
```

Comment:
@dvail Actually we don't need this type anymore because we're not using it anymore

Reply:
Cool thanks for taking a look. I removed that and the [only logic change](https://github.com/stackrox/stackrox/pull/5949/files#diff-d9bef30aa3fd8e6f401ac28d1f2484970f22ce021da71228acaf96fb08a75d8aR28) in the code that is impacted, as well as some old test code.